openedx/adg/lms/mysite/migrations/0002_auto_20210812_1050.py

```diff
@@ -0,0 +1,86 @@
+# Generated by Django 2.2.17 on 2021-08-12 10:50
+
+import django.core.validators
+from django.db import migrations, models
+import django.utils.timezone
+import model_utils.fields
+import openedx.adg.lms.mysite.helpers
```

Comment:
There is no helper file in PR. can you please verify that?

Reply:
Also, we shouldn't be creating 3 migration files. Revert these migrations and create a single migration file.